src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -179,6 +200,16 @@ class MediaBaseViewer extends BaseViewer {
             .catch(this.handleAssetError);
     }
 
+    /**
+     * Add event listeners to the media element related to loading of data
+     * @return {void}
+     */
+    addEventListenersForMediaLoad() {
+        this.mediaEl.addEventListener('canplay', this.stopBufferFillTimer);
```

Comment:
Should this be named something like `handleCanPlay`?

Reply:
👍